cypress/integration/multi-page.js

```diff
@@ -1,9 +1,18 @@
+const FIXED_DATE = new Date(2019, 1, 1).getTime()
+
 describe('multipage visual regression tests', () => {
-  it('Loads the multipage campaign correctly', function() {
+  beforeEach(function () {
+    cy.clock(FIXED_DATE);
+  });
 
+  it('Loads the multipage campaign correctly', function() {
     // Load the multipage campaign
     cy.visit(`/en/campaigns/multi-page/`);
 
+    // Give the browser a few seconds for JSX
+    // conversion to kick in.
+    cy.wait(10000);
```

Comment:
lol

Reply:
It's either that, or event-listen for the JSX to be done =D